scripts/difftime.py

```diff
@@ -1,9 +1,9 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 import readline  # noqa
 
 
 def parse_time(s):
-    h, m = map(int, s.strip().split(':'))
+    h, m = list(map(int, s.strip().split(':')))
```

Comment:
Ok, on the one had I understand what you're doing here, but on the other hand it feels like correcting Shakespeare to modern English with a red sharpie.

My own fault for naming the directory `scripts/` rather than `museum/`!